### PR TITLE
Update AWS/Azure READMEs with instructions for setting `cluster_version` and add missing permission for AWS

### LINKED ISSUE #682: Missing AWS IAM permission and link for supported GKE versions on AWS/Azure
Terraform template sets `cluster_version` but is quickly out of date due to regular releases and EoL versioning.
- need to add link and instructions for setting cluster_version in `AWS/terraform.tfvars` and `Azure/terraform.tfvars`

New required AWS IAM permission `iam:GetInstanceProfile` missing from template.
- add required permission to template

### PR DESCRIPTION
**Pull request by echyam**
- Update AWS/Azure READMEs with instructions for setting `cluster_version` with link to supported versions. 
- Add missing permission `iam:GetInstanceProfile` for AWS

### Fixes #682 
- If you still haven't done yet, then please do create an issue in the repository before opening this PR

### CODE REVIEW

File: anthos-multi-cloud/Azure/vars.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export CLUSTER_NAME=azure-cluster-fg
```
Comment (hankfreund) [inline anthos-multi-cloud/Azure/vars.sh:1]: Probably want to scrub personal project details from here.

Comment (echyam) [reply anthos-multi-cloud/Azure/vars.sh]: Yep, removed

**Review by hankfreund — APPROVED** [review]

**Pull request merged**